app/views/items/browse_facet.html.erb

```diff
@@ -42,21 +42,30 @@
     <div role="tabpanel"
       class="tab-pane <%= active?(facet_name, @browse_facet) %>" id="<%= id %>">
       <% if @res.key?(facet_name) %>
-        <ul class="list-unstyled">
-          <% @res[facet_name].each do |list| %>
-            <li class="row">
-              <%= link_to search_path("f" => ["#{facet_name}|#{list[0]}"]) do %>
-                <% text = list[0].blank? ? "No label" : list[0] %>
-                <div class="col-lg-5 col-md-6 col-sm-8 col-xs-10 browse-item">
-                  <span><%= text %></span>
-                </div>
-                <div class="col-md-1 col-xs-2">
-                  <span class="badge"><%= list[1] %></span>
-                </div>
-              <% end %>
-            </li>
-          <% end %>
-        </ul>
+        <table class="index_table">
+          <tbody>
+            <% @res[facet_name].each do |list| %>
+              <tr>
+                  <% text = list[0].blank? ? "No label" : list[0] %>
+                  <td class="index_num">
+                    <%= link_to search_path("f" => ["#{facet_name}|#{list[0]}"]) do %>
+                      <%= list[1] %>
+                    <% end %>
+                  </td>
+                  <td class="index_spacer">
+                    <%= link_to search_path("f" => ["#{facet_name}|#{list[0]}"]) do %>
+                      <span>&nbsp;</span>
```

Comment:
The `&nbsp;` causes an unpleasant small underline when mouse hovering the dotted line. I removed the `&nbsp;` but left the `<span>` locally and things still displayed well without the underline showing up. Looks good in Firefox and Chrome. Could y'all give that a test too?